src/system/PageLockModule/Api/LockingApi.php

```diff
@@ -24,6 +27,16 @@
 class LockingApi
 {
     /**
+     * Amount of required/opened accesses.
+     */
+    static $pageLockAccessCount = 0;
```

Comment:
why is it important that this be a static var/prop? I'm just trying to understand.